tests/tests/fee_tests.cpp

```diff
@@ -272,6 +272,8 @@ BOOST_AUTO_TEST_CASE(asset_claim_pool_test)
            }
         }
 
+        // can't claim pool because it is empty
+        GRAPHENE_REQUIRE_THROW( claim_pool( alice_id, aliceusd.id, _core(1), core_asset), fc::exception );
```

Comment:
I think this is unneeded, because there is already another test case below: "can't claim more than is available in the fee pool".

Although it doesn't harm to have it here right now, after fixed the issue above, the pool will no long be empty here so this test will fail.

Reply:
Please read comment above